doc/breakages.md

```diff
@@ -1,12 +1,22 @@
 # Minetest Major Breakages List
 
 This document contains a list of breaking changes to be made in the next major version.
+This list is largely advisory and items may be reevaluated once the time comes.
 
-* Remove attachment space multiplier (*10)
-* Remove player gravity multiplier (*2)
+* remove attachment space multiplier (*10)
+* remove space multiplier for models (*10)
+* remove player gravity multiplier (*2)
 * `get_sky()` returns a table (without arg)
 * `game.conf` name/id mess
 * remove `depends.txt` / `description.txt` (would simplify ContentDB and Minetest code a little)
-* rotate moon texture by 180°, making it coherent with the sun (see https://github.com/minetest/minetest/pull/11902)
+* rotate moon texture by 180°, making it coherent with the sun
+  * https://github.com/minetest/minetest/pull/11902
 * remove undocumented `set_physics_override(num, num, num)`
 * remove special handling of `${key}` syntax in metadata values
+* remove old_move
+* change physics_override `sneak` to disable the speed change and not just the node clipping
+  * https://github.com/minetest/minetest/issues/13699
+* migrate from player names to UUIDs, this would enable renaming of accounts and unicode player names (if desired)
```

Comment:
Regarding Unicode names, what about homoglyph attacks? Would the engine prevent them?

Reply:
That will be reevaluated if/when that is about to be implemented.